src/pages/Yat/Yat.tsx

```diff
@@ -102,26 +117,38 @@ export const Yat: React.FC = () => {
             <Row.Label>{translate('features.yat.usdcAddress')}</Row.Label>
             <Row.Value wordBreak='break-all'>
               {maybeYatUsdcAddress === null
-                ? 'loading'
+                ? 'Loading...'
                 : maybeYatUsdcAddress === ''
-                ? 'no usdc addy'
+                ? 'No USDC Address Found'
                 : maybeYatUsdcAddress}
             </Row.Value>
           </Row>
           <Row variant='vert-gutter'>
             <Row.Label>{translate('features.yat.ethAddress')}</Row.Label>
             <Row.Value wordBreak='break-all'>
               {maybeYatEthAddress === null
-                ? 'loading'
+                ? 'Loading...'
                 : maybeYatEthAddress === ''
-                ? 'no eth addy'
+                ? 'No ETH Address Found'
                 : maybeYatEthAddress}
             </Row.Value>
           </Row>
         </Summary>
-        <Button as={NavLink} to='/dashboard' colorScheme='blue' size='lg' mt={6}>
-          {translate('common.viewMyWallet')}
-        </Button>
+        {showMobileHandler ? (
+          <Button
+            as='a'
+            href={'shapeshift://yat/%F0%9F%A6%8A%F0%9F%9A%80%F0%9F%8C%88'}
```

Comment:
Are we hardcoding to Willy's yat? 🦊🚀🌈

![image](https://github.com/shapeshift/web/assets/97164662/ad958ec4-aa86-4025-979a-403e5943cbd7)